docs/build.md

```diff
@@ -18,13 +18,16 @@
   - iOS: `yarn ios`
   - Android: `yarn android`
   - Web: `yarn web`
+- If you are cloning or forking this repo as an open source developer, please check the tips below as well
 - Run e2e tests
   - Start in various console tabs:
     - `yarn e2e:server`
     - `yarn e2e:metro`
   - Run once: `yarn e2e:build`
   - Each test run: `yarn e2e:run`
 - Tips
+  - Make sure you copy the `.env.example` to `.env` and add the appropiate tokens (e.g. `SENTRY_AUTH_TOKEN` can be created on the Sentry dashboard). If this is not required, you can remove it from `eas.json` and `package.json`, as well as any mentions in the code.
```

Comment:
this doesn't fix for me, here is what I had to do: https://github.com/bluesky-social/social-app/issues/661